src/OpenSpaceToolkit/Physics/Data/Utilities.cpp

```diff
@@ -0,0 +1,62 @@
+#include <ctime>
+#include <filesystem>
+
+#include <OpenSpaceToolkit/Physics/Data/Utilities.hpp>
+#include <OpenSpaceToolkit/Physics/Time/DateTime.hpp>
+
+namespace ostk
+{
+namespace physics
+{
+namespace data
+{
+namespace utilities
+{
+
+using ostk::core::fs::File;
+
+using ostk::physics::time::Instant;
+using ostk::physics::time::Scale;
+using ostk::physics::time::DateTime;
+
+Instant getFileModifiedInstant(const File& aFile)
+{
+    if (!aFile.exists())
+    {
+        throw ostk::core::error::RuntimeError(
+            "Cannot get file modification time. File [{}] does not exist.", aFile.toString()
+        );
+    }
+
+    std::filesystem::path filePath = std::filesystem::path(std::string(aFile.getPath().getAbsolutePath().toString()));
+
+    const std::filesystem::file_time_type lastWriteTime = std::filesystem::last_write_time(filePath);
+
+    // TBI: this is an approximation. In c++ 20, there is a better way
```

Comment:
You could update physics to use C++ 20. I don't think there's any reason to not upgrade. We will definitely upgrade to 2023 as soon as it's released to use `std::fmt`

Reply:
Tried it. Looks like it's not a trivial upgrade from 17->20. Some stuff from Core breaks.

Reply:
Wow that's surprising. Well, that's a 2024 us problem :P